RPIPs/RPIP-42.md

```diff
@@ -1,113 +0,0 @@
-
-## Specification taking effect with Saturn 2
-- Update `reduced_bond` to 1.5 ETH
-
-
-## Penalizable offenses
-This portion of the RPIP SHALL be considered Living. It may be updated by a DAO vote.
-
-| Offense   | Penalty              | Added      | Updated    |
-|-----------|----------------------|------------|------------|
-| MEV theft | theft size + 0.2 ETH | 2024-03-29 | 2024-03-29 |
-
-## Implementation thoughts
-- When showing legacy node status, there is not a trivial way to get the node index for a given address. That said, the other direction is trivial using `RocketNodeManager.getNodeAt`, so the work can get moved off-chain. Eg, one can iterate across all possible node indices and then pass in the node index that matches the node's address; the smart contract can confirm the match to demonstrate legacy node status.
-
-## Rationale
-- Bond sizes were originally ideated per [prior work](../assets/rpip-42/bond_curves.md).
-  - `base_bond_array` is chosen to "sufficiently" dissuade MEV theft as a strategy
-  - `reduced_bond` is chosen to "sufficiently" guard against slashing or abandonment risks
-- Follow-on work was done in [discord](https://discord.com/channels/405159462932971535/1228753782402318427/1228914436924772352)
-  - The plots below show `base_bond_array`=[4, 8] and `reduced_bond`=1.5. As we can see, MEV theft always increases yield and the impact is heightened at low commission. The reality is that we've seen very little of this type of behavior. We may have to change our approach if we see MEV theft increase or if we wish to support NO commission share under 2.5%.
-  - A moderate step would be to change `base_bond_array` to a curve that reduces MEV theft advantage in the current context (commission, MEV landscape...) at the cost of user complexity, eg `[4.2, 6.8. 9.2. 11.4. 13.5. 15.5. 17.4]`
-  - A larger step would be to pass EL rewards to NOs and charge them for the benefit. See eg: <https://github.com/Valdorff/rp-thoughts/tree/main/leb_safety#negative-commission-aka-assign-execution-layer-rewards-to-nos> or <https://dao.rocketpool.net/t/reimagining-large-block-theft/2146>
-- The express queue is meant to favor (a) existing NOs and (b) small NOs. The end goal in both cases is to support multiple values enshrined in [RPIP-23](RPIP-23.md) (the pDAO charter): decentralization, protocol safety, and the health of the Ethereum network.
-  - The `express_queue_tickets_base_provision` is enough to get started, and currently matches the length of `base_bond_array`
-  - The tickets from `(bonded ETH in legacy minipools)/4` are enough to fully migrate to 4-ETH deposits during Saturn 1 using the express queue OR to partly migrate to 1.5-ETH deposits after Saturn 2
-  - It's worth emphasizing that the tickets stick around -- ie, a node operator joining during a time when we don't have an NO queue (ie, when RP has an immediate need for NO supply) gets to keep their express queue benefit for a later time if they wish
```

Comment:
This should be in the specification section, imo.

Reply:
I don't think it clarifies that they stick around? I guess that's the default as it doesn't specify an expiry or anything, but still. Either way, this paragraph isn't a rationale item, it's not explaining _why something is_ the way it is, it's explaining _what something is_.